service/create/service.go

```diff
@@ -454,6 +449,64 @@ func (s *Service) Boot() {
 						return
 					}
 
+					// Create Ingress Load Balancer.
+					lbInput = LoadBalancerInput{
+						Name:        cluster.Spec.Cluster.Kubernetes.IngressController.Domain,
+						Clients:     clients,
+						Cluster:     cluster,
+						InstanceIDs: workerIDs,
+						PortsToOpen: []int{
+							httpPort,
```

Comment:
I'm not sure I understand. What are the `InstancePort`s and `LoadBalancerPort`s we need?